index.html

```diff
@@ -9,37 +9,160 @@
                 </ul>
-            </nav>
-        </section>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+          <div class="second-card">
+            <div class="card-image"><img src="images/secondCover.png" alt="SecondCover-Image"></div>
+            <div class="card-content">
+              <article>
+                <h1>Multi-Post <br>Stories</h1>
+                <div class="card-time">
+                  <p class="darken">CANOPY •&nbsp;</p>
+                  <p> Back End Dev •&nbsp;</p>
+                  <p> 2015</p>
+                </div>
+                <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="card-list">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascipt</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+          <div class="third-card">
+            <div class="card-image"><img src="images/ThirdCover.png" alt="ThirdCover-Image"></div>
+            <div class="card-content">
+              <article>
+                <h1>Tonic</h1>
+                <div class="card-time">
+                  <p class="darken">CANOPY •&nbsp;</p>
+                  <p> Back End Dev •&nbsp;</p>
+                  <p> 2015</p>
+                </div>
+                <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="card-list">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascipt</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+          <div class="fourth-card">
+            <div class="card-image"><img src="images/fouthCover.png" alt="FourthCover-Image"></div>
+            <div class="card-content">
+              <article>
+                <h1>Multi-Post <br>Stories</h1>
+                <div class="card-time">
+                  <p class="darken">CANOPY •&nbsp;</p>
+                  <p> Back End Dev •&nbsp;</p>
+                  <p> 2015</p>
+                </div>
+                <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="card-list">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascipt</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+        </div>
+        <div class="about-section">
+          <section class="about-content">                      
+              <h1>About Myself</h1>
```

Comment:
- [ ] [REQUIRED] You did a very good job here but this is not the best pactice as the `section` tag does not come after the `div`. Therefore he container of the "About myself" section should be a `section` tag.

- [ ] [OPTIONAL] I would advise you to reconsider the remaining sections as well which will be great for your Porfolio project.

Reply:
done